apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/QueryOptions.cs

```diff
@@ -28,6 +28,11 @@ public sealed class QueryOptions : IEquatable<QueryOptions>
         /// </summary>
         public string OptimizerVersion { get => Proto.OptimizerVersion; }
 
+        /// <summary>
+        /// The query optimizer statistics package configured in the options.
+        /// </summary>
+        public string OptimizerStatisticsPackage { get => Proto.OptimizerStatisticsPackage; }
```

Comment:
I'm not sure why I didn't spot this before (for OptimizerVersion) but this can be simplified to:

```csharp
 public string OptimizerStatisticsPackage => Proto.OptimizerStatisticsPackage;
```

(And that's the more idiomatic representation.)

Reply:
Much nicer. Done.